composer.json

```diff
@@ -33,13 +33,13 @@
         "files": ["src/functions.php"]
     },
     "require-dev": {
-        "zendframework/zend-form": "<2.8",
         "squizlabs/php_codesniffer": ">=2.7",
-        "phpunit/phpunit": ">=4.8.35 <=5.6"
+        "phpunit/phpunit": "^8 || ^9"
     },
     "autoload-dev": {
         "psr-4": {
-            "SwaggerTests\\": "tests/"
+            "SwaggerTests\\": "tests/",
+            "AnotherNamespace\\": "tests/Fixtures/AnotherNamespace"
```

Comment:
Noit sure if this is really needed seeing that the fixtures do not properly use PSR-4 namespaces.

Reply:
The fixtures use psr-4 namespace autoloading corrctly. Just with strange namespaces (\SwaggerTests, \AnotherNamespace). So I think this is needed to load the annotation examples.

Reply:
Hmm, I would argue that
```
namespace SwaggerFixtures;
```
is not a correct namespace according to PSR-0/PSR-4 as it does not match the folder name at all.

Nevertheless, that's not really important for the fixtures so all is good.